src/pkg/repository/repository.go

```diff
@@ -318,6 +319,11 @@ func (r repository) NewBackupWithLookup(
 		return operations.BackupOperation{}, errors.Wrap(err, "resolving resource owner details")
 	}
 
+	// Exchange and OneDrive need to maintain the user PN as the ID until we're ready to migrate
+	if sel.PathService() != path.SharePointService {
```

Comment:
It does seem like there's some SP stuff being done by SDK consumers. They've always been using IDs right?

Reply:
Yup!